README.rst

```diff
@@ -6,6 +6,35 @@
 :Homepage: http://engineering.odl.mit.edu
 :License: BSD
 
+PyLmod provides a Python library to access the MIT Learning Modules web
+service (described below). PyLmod was created to support
+MIT's use of OpenedX for residential courses, but the library is open
+source to enable easier access to that service for Python application
+developers at MIT. PyLmod encapsulates the Learning Modules web service
+making it more pythonic and easier to incorporate into Python applications.
+
+The MIT Learning Modules web service, maintained by MIT Information
+Systems and Technologies (IS&T), exposes an API to MIT systems of
+record for classes, students, and grades. Its documentation for gradebook
+and membership is available at these links.
+
+Gradebook module
+    `https://learning-modules-test.mit.edu/service/gradebook/doc.html
```

Comment:
I think it would be nicer if you just linked the `Gradebook module` (and `Membership module` below) text instead of the redundant URL, and maybe put these in a list